src/jloptions.c

```diff
@@ -191,6 +193,7 @@ JL_DLLEXPORT void jl_parse_opts(int *argcp, char ***argvp)
         { "check-bounds",    required_argument, 0, opt_check_bounds },
         { "output-bc",       required_argument, 0, opt_output_bc },
         { "output-unopt-bc", required_argument, 0, opt_output_unopt_bc },
+        { "output-jit-bc",   required_argument, 0, opt_output_jit_bc },
```

Comment:
Since this and the `--output-unopt-bc` are for developer debugging purposes only, I think I'll remove it from the help text and document it in the devdocs.